docs/htmlapps.md

```diff
@@ -48,3 +62,26 @@ methods for creating files within a zip file.
 See the source code:
 [ricecooker/utils/html_writer.py](https://github.com/learningequality/ricecooker/blob/master/ricecooker/utils/html_writer.py#L5)
 
+
+
+Static assets download utility
+------------------------------
+
+We have a handy function for fetching all of a webpage's static assets (JS, CSS, images, etc.), so that, in theory, you could scrape a webpage and display it in Kolibri exactly as you see it in the website itself in your browser. See:
+
+- the source: [`ricecooker.utils.downloader.download_static_assets()`](https://github.com/learningequality/ricecooker/blob/428bfde98e0f76310eccd367886aebe62cd9ae5a/ricecooker/utils/downloader.py#L129)
+- [example usage in a simple app: MEET chef](https://github.com/learningequality/sushi-chef-MEET/blob/425327ad552f9f25f582a2057048f6d4475382c1/chef.py#L205), which comprises articles with text and images
+- [example usage in a complex app: Blockly Games chef](https://github.com/learningequality/sushi-chef-blockly-games/blob/270e8bc620be0ed883f40e2739878db54f7243b7/chef.py#L193), an interactive JS game with images and sounds
+
+
+
+Starter template
+----------------
+
+We also have a [starter template](https://github.com/learningequality/html-app-starter) for apps, particularly helpful for displaying content that's mostly text and images, such as articles. It applies some default styling on text to ensure readability, consistency, and mobile responsiveness.
```

Comment:
Are the css classes documented either here or on the starter template repo?

Reply:
Good point. There is a basic usage example in the README here: https://github.com/learningequality/html-app-starter#test-it-out
but we'll need to add more docs + usage example when we merge the template into ricecooker.

(currently template is in still in user-testing mode; that's why we keep it in a separate repo)